src/components/Reports/GDPRPermissions/PermissionExpandableRow.js

```diff
@@ -63,7 +63,7 @@ const PermissionExpandableRow = ({
               <button
                 type="button"
                 onClick={() => setExpanded(!expanded)}
-                className={`dp-expand-results ${expanded && 'dp-open-results'}`}
+                className={`dp-expand-results${expanded ? ' dp-open-results' : ''}`}
```

Comment:
I understand what you did, but, even when a extra space will be rendered I prefer the following approach (if it works fine):

```suggestion
                className={`dp-expand-results ${expanded ? 'dp-open-results' : ''}`}
```

In my opinion it is cleaner. What do you think?

Reply:
I agree with the proposed approach. It looks cleaner.